target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -85,11 +85,12 @@ c20i)
 	ucidef_set_led_wlan "wlan" "wlan" "$board:blue:wlan" "phy0radio"
 	;;
 c50)
-	ucidef_set_led_default "power" "power" "$board:blue:power" "0"
-	ucidef_set_led_netdev "lan" "lan" "$board:blue:lan" "eth0.2"
-	set_usb_led "$board:blue:usb"
-	ucidef_set_led_wlan "wlan2g" "wlan2g" "$board:blue:wlan2g" "phy1radio"
-	ucidef_set_led_wlan "wlan5g" "wlan5g" "$board:blue:wlan5g" "phy0radio"
+	ucidef_set_led_default "power" "power" "$board:green:power" "0"
```

Comment:
Which doesn't mean it was correct before...

According to the change done by @heinzek a few seconds ago the power led is ACTIVE_LOW instead of the former ACTIVE_HIGH.

@heinzek please add a few words about the ACTIVE_HIGH => ACTIVE_LOW change to the commit message to make clear this change was intentional.